exercises/concept/lucky-numbers/.meta/config.json

```
@@ -1,6 +1,6 @@
 {
-  "blurb": "TODO: add blurb for lucky-numbers exercise",
-  "authors": ["shubhsk88"],
+  "blurb": "Practice type conversion while helping your friend Kojo with his website.",
```

I know blurbs are like "whatever", but... this one feels a bit weird to me because it doesn't have any connection to the exercise's name. Maybe:
```suggestion
  "blurb": "Practice type conversion while helping your friend Kojo with his website www.fun-with-numbers.com.",
```

---

fixed, I do think blurbs are important